x/execution/handler.go

```diff
@@ -30,7 +26,7 @@ func NewHandler(k Keeper) sdk.Handler {
 func handleMsgCreateExecution(ctx sdk.Context, k Keeper, msg MsgCreateExecution) (*sdk.Result, error) {
 	exec, err := k.Create(ctx, msg)
 	if err != nil {
-		return nil, sdkerrors.Wrap(errors.ErrUnknown, err.Error())
+		return nil, err
```

Comment:
why removing them here?

Reply:
before the error is return from a keeper function that should always already returned a wrap error.

Reply:
It's a bit weird to have some errors that are returned as error that don't need wrapping and some that need. Would love to find a way to fix that, maybe a specific check might be good or a specific type.

Anyway, this is not blocking and can be improved later

Reply:
I checked to see if I could check if an error is already wrapped but couldn't find a nice way.